scanners/android/android.go

```diff
@@ -46,73 +48,108 @@ func (scanner *Scanner) DetectPlatform(searchDir string) (_ bool, err error) {
 
 	log.TInfof("Searching for android files")
 
-	scanner.ProjectRoots, err = walkMultipleFileGroups(searchDir, projectFiles, skipDirs)
+	projectRoots, err := walkMultipleFileGroups(searchDir, projectFiles, skipDirs)
 	if err != nil {
-		return false, fmt.Errorf("failed to search for build.gradle files, error: %s", err)
+		return false, nil, fmt.Errorf("failed to search for build.gradle files, error: %s", err)
 	}
 
-	log.TSuccessf("Platform detected")
-
+	log.TPrintf("%d android files detected", len(projectRoots))
 	for _, file := range projectFiles {
```

Comment:
should not this iterate in `projectRoots` instead of the `projectFiles` ?

Reply:
Indeed.